tube_tipper.scad

```diff
@@ -40,7 +40,7 @@ module tube_tipper(number_tubes = 5) {
         for (x = [0, 1]) {
           translate([-wall_thickness,
             (y - (number_tubes+1)/2) * (tube_diameter + tube_spacing),
-            -0.5 * (tube_spacing + tube_diameter) + (0.5 + x) * (tube_diameter + tube_z_stretch + tube_spacing)])
+            -0.5 * (tube_spacing + 0.75 * tube_diameter) + (0.5 + x) * (tube_diameter + tube_z_stretch + tube_spacing)])
```

Comment:
it seems this translate does not yet move the bottom row down more, wasn't this already part of the last printing run? 
sidenote: I think for the 20degree angle to keep working you should only move the bottom holes lower, the top ones must stay where they are, right?

Reply:
Moving the bottom row down was part of the last test, I think I need to move it a little back up so that the bottom of the hole is sitting right at the base.

On the sidenote: yes, that is how angles work, fixing now.